--- !ditz.rubyforge.org,2008-03-06/issue 
title: "test262 interfacing with remote running: use of stderr, exit code"
desc: |-
  E.g. SyntaxError is visible in stderr only (same in Rhino), but a non-zero
  exit code is not preserved by remote execution through ssh.
type: :task
component: duk
release: v2.0
reporter: sva <sami.vaarala@iki.fi>
status: :unstarted
disposition: 
creation_time: 2014-02-21 20:51:01.123430 Z
references: []

id: 60a78f5758f3d78601808de41bd5ac78da990eb2
log_events: 
- - 2014-02-21 20:51:01.290795 Z
  - sva <sami.vaarala@iki.fi>
  - created
  - ""
- - 2014-03-28 10:26:27.275801 Z
  - Sami Vaarala <sami.vaarala@iki.fi>
  - assigned to release v0.11 from v0.10
  - ""
- - 2014-05-07 21:10:53.214427 Z
  - Sami Vaarala <sami.vaarala@iki.fi>
  - assigned to release v0.12 from v0.11
  - ""
- - 2014-09-09 11:31:10.151191 Z
  - Sami Vaarala <sami.vaarala@iki.fi>
  - assigned to release v1.1 from v0.12
  - ""
- - 2014-11-02 00:14:48.157285 Z
  - Sami Vaarala <sami.vaarala@iki.fi>
  - assigned to release v2.0 from v1.1
  - ""
